model/task/db.go

```diff
@@ -2962,3 +2962,46 @@ func getGenerateTasksEstimation(ctx context.Context, project, buildVariant, disp
 
 	return results, nil
 }
+
+// GetLatestTaskFromImage retrieves the latest task from all the distros corresponding to the imageID.
+func GetLatestTaskFromImage(ctx context.Context, imageID string) (*Task, error) {
+	distros, err := distro.GetDistrosForImage(ctx, imageID)
+	if err != nil {
+		return nil, errors.Wrap(err, "retrieving distros from imageID")
+	}
+	distroNames := make([]string, len(distros))
+	for i, d := range distros {
+		distroNames[i] = d.Id
+	}
+	if len(distroNames) == 0 {
+		return nil, errors.New("No distros found for image.")
```

Comment:
nit. I looked at other instances of `errors.New` in our code and it seems like they follow this format (no capitalization, no punctuation)
```suggestion
		return nil, errors.New("no distros found for image")
```